language-server/modules/langserver-compiler/src/main/java/org/ballerinalang/langserver/compiler/config/DataMapperConfig.java

```diff
@@ -0,0 +1,24 @@
+package org.ballerinalang.langserver.compiler.config;
```

Comment:
Missing license agreement

Reply:
Added license agreement

Reply:
It seems the license header is not properly added. The last part is missing and also new lines are missing. Refer [1] for example.

[1] https://github.com/ballerina-platform/ballerina-lang/blob/master/tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/preloading/LSClientConfig.java